portal_host.properties

```diff
@@ -0,0 +1 @@
+172.31.15.208
```

Comment:
I'd suggest to not check in this file and ask user to create it if it doesn't exist. 

This can be done by add a check in create_portal_host.sh to verify portal_host.properties exists, and if it doesn't, prompt an error message and ask the user to create it. this is the same way slaves.in file is used.

Reply:
Added

Reply:
thanks!